src/bridge/cockpithttpstream.c

```diff
@@ -726,11 +750,9 @@ send_http_request (CockpitHttpStream *self)
   JsonObject *headers;
   const gchar *header;
   const gchar *value;
-  GList *request = NULL;
-  GList *names = NULL;
+  g_autoptr(GList) names = NULL;
```

Comment:
is this even being used anywhere?

Reply:
It's used for storing the list of `headers` option's members.